storage/objects.go

```diff
@@ -205,6 +256,48 @@ func (s *ObjectsClient) Delete(ctx context.Context, input *DeleteObjectInput) er
 	return nil
 }
 
+// GetMpuInput represents parameters to a GetMpu operation
+type GetMpuInput struct {
+	PartsDirectoryPath string
+}
+
+type GetMpuHeaders struct {
+	ContentLength int64  `json:"content-length"`
+	ContentMd5    string `json:"content-md5"`
+}
+
+type GetMpuOutput struct {
+	Id             string        `json:"id"`
+	State          string        `json:"state"`
+	PartsDirectory string        `json:"partsDirectory"`
+	TargetObject   string        `json:"targetObject"`
+	Headers        GetMpuHeaders `json:"headers"`
+	NumCopies      int64         `json:"numCopies"`
+	CreationTimeMs int64         `json:"creationTimeMs"`
+}
+
+func (s *ObjectsClient) GetMultipartUpload(ctx context.Context, input *GetMpuInput) (*GetMpuOutput, error) {
+	return getMpu(*s, ctx, input)
+}
+
+type ListMpuPartsInput struct {
+	Id string
+}
+
+type ListMpuPart struct {
```

Comment:
Do we want to add all of the fields that are returned as part of an object listing like (content-type, modified time, etc)?

Reply:
So the only documentation I've seen on this is [here](https://github.com/joyent/rfd/blob/master/rfd/0065/README.md#5-list-parts) and the rest of this was derived from the behavior of the existing node client and a reading of the muskie code. Am I missing a more detailed list of what is returned in the response?

Reply:
Here is what was returned from one of the requests that I did:

```json
{
  "name": "0",
  "etag": "09de2770-0684-40e1-82d5-c0f881c03a24",
  "size": 5242880,
  "type": "object",
  "contentType": "application/octet-stream",
  "contentMD5": "vfGT/owyBJ6/PjRl/A4hyQ==",
  "mtime": "2019-01-03T01:55:56.384Z",
  "durability": 2
}
```

Reply:
Do you know if that is stable?

Reply:
I think there is a broad issue with documentation for MPU from the Manta side of things. I'll create a new ticket to get all of that updated and I'll create an issue to update our response values for MPU calls once that has happened.